### DISCUSSION TITLE
ECLISPSE PLUGIN OFFLINE INSTALL ?

### DISCUSSION THREAD

**Discussion by RadFromOrange** [hallvard/plantuml#159] [Q&A]
Hello , is there a way to install plantuml eclipse plugin offline from a zip or jar ?

Thanx,

**Comment by dbill2011** [comment]
Hello,

Did you solve your problem ?

I tried to load PlantUML bundle into eclipse/dropins directory, using latest version 1.1.27 donwloaded from hallvard plantuml site but Eclipse Oxygen is just writing the following error message:

No repository found at file:<root dir>/eclipse/bundles

Seems that donwloaded package is not adapted to dropins eclipse directory.

Best regards

**Reply by RadFromOrange** [reply]
Hi, Sorry no i have convinced my team that this is a useless ( for our use case) and ugly tools used in 2000's .

Switched to draw.io and Figjam for professional result.

Good luck
